index.rkt

```diff
@@ -360,6 +360,10 @@ compiler, and @italic{that} syntax is evaluated:
 (values "i" "am" "backwards")
 ]
 
+@margin-note{The first argument of @racket[syntax->datum] contains the lexical
+context information that we want to associate with the @racket[syntax]
+outputted by the transformer.}
+
 @; ----------------------------------------------------------------------------
```

Comment:
This is a nice explanation. I like that it's a margin-note not in the main flow -- because at this very early stage "lexical context" might be confusing or overwhelming?